dotcom-rendering/src/components/MoreGalleries.stories.tsx

```diff
@@ -23,4 +24,41 @@ export const MoreGalleries = {
 			theme: Pillar.Culture,
 		},
 	},
+	parameters: {
+		chromatic: {
+			viewports: [breakpoints.mobile, breakpoints.desktop],
+		},
+	},
```

Comment:
Just a heads up, the `viewports` parameter is deprecated[^1] in favour of the modes API. [Here are some docs](https://main--63e251470cfbe61776b0ef19.chromatic.com/index.html?path=/docs/writing-stories--docs#chromatic-and-modes) on our use of modes in DCAR.

[^1]: https://www.chromatic.com/docs/legacy-viewports/#viewports-legacy-storybook-api

Reply:
Thanks for the heads up! I'll make sure to use `modes` in the future.